app/scripts/modules/amazon/src/instance/details/utils.ts

```diff
@@ -17,7 +17,12 @@ export const applyHealthCheckInfoToTargetGroups = (
     if (metric.type === 'TargetGroup') {
       metric.targetGroups.forEach((tg: ITargetGroup) => {
         const group = targetGroups[tg.name] ?? ({} as ITargetGroup);
-        const port = group.healthCheckPort && isNumber(group.healthCheckPort) ? group.healthCheckPort : group.port;
+        let port = group.port;
```

Comment:
I think this could be simpler and easier to understand the intent like this:
```
const useTrafficPort = group.healthCheckPort === 'traffic-port' || isNil(group.healthCheckPort);
const port = useTrafficPort ? group.port : group.healthCheckPort;
```

I think we don't really need to parseInt because we're only converting back to a string in a subsequent line
```
        tg.healthCheckPath = `:${port}${group.healthCheckPath}`;
```